components/test_pd/src/mocker/retry.rs

```diff
@@ -36,6 +36,12 @@ impl Retry {
     }
 }
 
+impl Default for Retry {
+    fn default() -> Self {
+        Self::new(0)
+    }
+}
+
```

Comment:
This `Deafult` implementation looks questionable to me. If the user does not need a retry, they should use `NotRetry`. "Retry 0 times" seems not a good default...

And because `Retry::new` takes an argument, clippy should not complain about it. I prefer to just remove this implementation.

Reply:
> And because Retry::new takes an argument, clippy should not complain about it. I prefer to just remove this implementation.

You mean we can just let clippy omit this implementation?

> This Deafult implementation looks questionable to me. If the user does not need a retry, they should use NotRetry. "Retry 0 times" seems not a good default
Shall we change 0 to a reasonable value ?

Reply:
Will clippy complain if we don't implement `Default` for `Retry`?

Reply:
> You mean we can just let clippy omit this implementation?

Were you complained by clippy about missing default for `Retry`? I just remove this implementation and clippy does not give a warning.

Reply:
Yes, Clippy will output warning information. i wander if exists some way to make this method as an exception.

Reply:
```
#[allow(clippy::xxx)]
```

If clippy still gives a warning, use the above marker to ignore check for a single line / struct.